inventory/sample/group_vars/k8s_cluster/addons.yml

```diff
@@ -161,6 +161,12 @@ cert_manager_enabled: false
 #   -----END CERTIFICATE-----
 # cert_manager_leader_election_namespace: kube-system
 
+# cert_manager_dns_policy: "None"
```

Comment:
Could you update here to `ClusterFirst` to follow the default value?

Reply:
Done.